apps/passport-server/src/services.ts

```diff
@@ -27,6 +28,10 @@ export async function startServices(
   await startTelemetry(context);
   instrumentPCDs();
 
+  const verificationPromiseCache = new LRUCache<string, Promise<boolean>>({
+    max: 1000
+  });
```

Comment:
i think i would prefer for this to live in `issuanceService`, and for frog crypto service to take an issuance service as a parameter, and for `issuanceService` to expose a method to get the promise cache so that `frogCryptoService` can ask for it inside of its constructor.

tl;dr the way you're doing it is not strictly incorrect, it just doesn't fit within my model of the structure of the server.